vms/evm/subscriber.go

```diff
@@ -33,33 +36,76 @@ var (
 	ErrInvalidLog = errors.New("invalid warp message log")
 )
 
+// The filter query used to match logs emitted by the Warp precompile
+var warpFilterQuery = interfaces.FilterQuery{
+	Topics: [][]common.Hash{
+		{warp.WarpABI.Events["SendWarpMessage"].ID},
+		{},
+		{},
+	},
+	Addresses: []common.Address{
+		warp.ContractAddress,
+	},
+}
+
 // subscriber implements Subscriber
 type subscriber struct {
-	nodeURL string
-	log     chan vmtypes.WarpLogInfo
-	evmLog  <-chan types.Log
-	sub     interfaces.Subscription
+	nodeWSURL  string
+	nodeRPCURL string
+	chainID    ids.ID
+	log        chan vmtypes.WarpLogInfo
```

Comment:
Not new in this PR, but would it be more clear if we named this `logChan`?

Reply:
Good idea. Changed.